x/ecocredit/params.go

```diff
@@ -87,5 +87,5 @@ func NewParams(creditClassFee sdk.Coins, creditTypes []*CreditType) Params {
 }
 
 func DefaultParams() Params {
-	return NewParams(sdk.NewCoins(sdk.NewCoin(sdk.DefaultBondDenom, DefaultCreditClassFeeTokens)), []*CreditType{})
+	return NewParams(sdk.NewCoins(sdk.NewCoin(sdk.DefaultBondDenom, DefaultCreditClassFeeTokens)), []*CreditType{{Name: "carbon", Units: "kg", Precision: PRECISION}})
```

Comment:
Not sure if this is ideal. This is the easiest path forward to appease the new integration tests, as there wasn't really a simple way to set the params with those tests. 

Would this line have to be reverting before build? not really sure how params get shipped with upgrades..

Reply:
thoughts ? - @aaronc @blushi @clevinson

Reply:
Maybe this is okay. But shouldn’t it be tons not kg?

Reply:
ok sure - just had it as a placeholder , switched to tons. I opened an issue here https://github.com/regen-network/regen-ledger/issues/457 to discuss what to do about default params

Reply:
Looks like this should be fine :)